app/actors/hydranorth/generic_file/actor.rb

```diff
@@ -49,6 +49,19 @@ def create_metadata_with_resource_type(batch_id, resource_type)
         create_metadata(batch_id)
       end
 
+      def destroy
+        byebug
+        generic_file.destroy
+        FeaturedWork.where(generic_file_id: generic_file.id).destroy_all
+
+        ark_identifier = Ezid::Identifier.find(Ezid::Client.config.default_shoulder + @generic_file.id)
```

Comment:
Likewise to https://github.com/ualbertalib/HydraNorth/pull/933/files#diff-c797d2271933d72ac35d1b2fb44ff60dR80 - why not use @generic_file.ark_id?